### DISCUSSION TITLE
关于编译和链接

### DISCUSSION THREAD

**Discussion by longlongvip** [xmake-io/xmake#4423] [General]
- [程序员的自我修养：链接、装载与库](https://awesome-programming-books.github.io/others/%E7%A8%8B%E5%BA%8F%E5%91%98%E7%9A%84%E8%87%AA%E6%88%91%E4%BF%AE%E5%85%BB%EF%BC%9A%E9%93%BE%E6%8E%A5%E3%80%81%E8%A3%85%E8%BD%BD%E4%B8%8E%E5%BA%93.pdf)
- Windows 编译与链接 官方文档 https://learn.microsoft.com/zh-cn/cpp/build/projects-and-build-systems-cpp?view=msvc-170
- issue太多了
- 我是菜鸟

**Comment by waruqi** [comment]
?

**Reply by longlongvip** [reply]
大家包括我遇到的问题太多的原因还是对编译和链接理解不深刻

**Reply by longlongvip** [reply]
对编译和链接理解不深刻的话，啥好工具对他都不好使

**Comment by star-hengxing** [comment]
虽然但是，个人聊骚/分享可以放在博客或者其他平台= =
